### PR TITLE
Fast MurmurHash2 on Python 3

### PR DESCRIPTION
**Pull request by twm**
Swap in pyhash for Murmur to get C hashing on Python 3.

### CODE REVIEW

**Comment by twm** [comment]
It looks like all the PyPy builds are failing because:

> **Notes** pyhash only support pypy v6.0 or newer, please [download and install](https://pypy.org/download.html) the latest pypy.

**Comment by twm** [comment]
(My present strategy for this PR is to leave it open until Travis adds pypy 6.0.)

File: afkak/partitioner.py [hunk 1]
```diff
@@ -1,53 +1,33 @@
 # -*- coding: utf-8 -*-
-# Copyright (C) 2015 Cyan, Inc.
-# Copyright 2017 Ciena Corporation
-
-import logging
+# Copyright 2015 Cyan, Inc.
+# Copyright 2017, 2018 Ciena Corporation
```
Comment (rthille) [inline afkak/partitioner.py]: Why?

Comment (rthille) [reply afkak/partitioner.py]: Oh, never mind. Brainfart. I didn't clue into the `import logging` being removed and thought you were just moving it below that line.

**Pull request merged**